src/main/java/com/umbr3114/controllers/DropController.java

```diff
@@ -56,8 +63,33 @@
         dropCollection = new CollectionFactory<DropModel>(ServiceLocator.getService().dbService(), DropModel.class).getCollection();
         dropCollection.save(drop);
 
-        return new GeneralResponse(HttpStatus.OK_200, "successful");
+        return "okay";
+    });
+
+    public static Route updateDrop = ((request, response) -> {
+        String dropId;
+        String description = null;
+        RequestParamHelper helper = new RequestParamHelper(request);
+        JacksonMongoCollection<DropModel> update;
+        dropId = helper.valueOf("dropId");
+        update = new CollectionFactory<DropModel>(ServiceLocator.getService().dbService(), DropModel.class).getCollection();
+        update.updateOne(eq("_id", new ObjectId("dropId")), combine (set("title", 1), set("topic", ""), currentDate("lastModified")));
+        if (description == null || description.isEmpty());
+        halt(HttpStatus.NOT_FOUND_404, "did not update");
+        return "something";
+
+
+    });
+
+    public static Route deteleDrop = ((request, response) -> {
+        String drop_Id;
+        RequestParamHelper helper = new RequestParamHelper(request);
+        JacksonMongoCollection<DropModel> delete;
+        drop_Id = helper.valueOf("drop_id");
+       delete = new CollectionFactory<DropModel>(ServiceLocator.getService().dbService(), DropModel.class).getCollection();
+       delete.deleteOne(eq("_id", new ObjectId(drop_Id)));
 
+       return "something";
```

Comment:
According to the issue, the route was supposed to return JSON. This route currently returns "something"

Take a look at the GeneralResponse class and how it is used. (it is used in the master version of DropModel, PostModel, AuthRoutes)